LBS/main/auth/auth_guard.py

```diff
@@ -22,11 +23,27 @@ def decorated_function(*args, **kwargs):
             try:
                 user_data = check_jwt()
             except Exception as e:
+                log_action(__name__, str(e), route_function.__name__, "", "", "warning")
+
                 return jsonify({"message": f"{e}", "status": 401}), 401
+
             # Authorization gate
             if role and role not in user_data["roles"]:
+                log_action(
+                    __name__,
+                    "Invalid Role Authorization required",
+                    route_function.__name__,
+                    "",
+                    "",
+                    "warning",
+                )
                 return (
-                    jsonify({"message": "Authorization required.", "status": 403}),
```

Comment:
Do not change the code style when you work on a feature. You can't even imagine how hard it is to review a commit with tons of pointless change. Specifically, when there is 15 commits.